### PR TITLE
Update README.md to include UCLan legal info

### PR DESCRIPTION
**Pull request by Sam5065**
Updated README.md to include copyright info for UCLan logo (info used from [https://www.uclan.ac.uk/legal/copyright](url))

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Reads fine!

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ MazeMap uses campus IDs to show campuses around the world, UCLan campus ID is 36
 
 <img width="1331" alt="Screenshot 2023-11-15 at 22 25 04" src="https://github.com/insert-professional-sounding-name-here/insert-professional-sounding-name-here.github.io/assets/20979204/4e579d30-1745-43f5-94a7-946534bf6d17">
 
-#### This repo is only for educational use for a university assignment and not to be used in a real-world situation or for commercial purposes. All copyrights reserved to their respective owners. MazeMap and related services are provided by MazeMap AS
+#### This repo is only for educational use for a university assignment and not to be used in a real-world situation or for commercial purposes. All copyrights reserved to their respective owners. MazeMap and related services are provided by MazeMap AS. The the UCLan logo is owned by the University of Central Lancashire.
```
Comment (Sam5065) [inline README.md:27]: Should keep this the same or Should it be more like 

> owned by the University of Central Lancashire all rights revered

**Review by Sam5065 — COMMENTED** [review]
@BOConnor1 Ive added a comment about the UCLan logo legal info, would like to hear your thoughts on it

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:27]: Possibly both? "The UClan logo is owned by University of Central Lancashire, all rights reserved"?

**Pull request merged**